src/purchases.ts

```diff
@@ -250,7 +256,8 @@ export default class Purchases {
    * @param {String} apiKey RevenueCat API Key. Needs to be a String
    * @param {String?} appUserID An optional unique id for identifying the user. Needs to be a string.
    * @param {boolean} [observerMode=false] An optional boolean. Set this to TRUE if you have your own IAP implementation and want to use only RevenueCat's backend. Default is FALSE.
-   * @param {boolean} [usesStoreKit2IfAvailable=false] An optional boolean. iOS-only. Set this to TRUE to enable StoreKit2 on compatible devices.
+   * @param {boolean} [usesStoreKit2IfAvailable=false] DEPRECATED. An optional boolean. iOS-only. Defaults to FALSE. Setting this to TRUE will enable StoreKit2 on compatible devices. 
+   * Simply remove this parameter to let RevenueCat decide for you which StoreKit implementation to use.
```

Comment:
"Remove" seems weird, since we're asking them not to use it.
Maybe "We recommend not using this parameter to let..."

Reply:
shipping for now, let me know if you have any further suggestions and I can open up a new PR